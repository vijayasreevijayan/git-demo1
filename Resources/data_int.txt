1,13,19
9, 12, 8, 7
3, 5, 6, 7, 9