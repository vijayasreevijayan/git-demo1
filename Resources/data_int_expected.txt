data_int.txt
Line 1 average 11 standard deviation 9.16515
Line 2 average 9 standard deviation 2.16025
Line 3 average 6 standard deviation 2.23607